ignite/cmd/network_chain_launch.go

```diff
@@ -19,9 +19,31 @@ const (
 func NewNetworkChainLaunch() *cobra.Command {
 	c := &cobra.Command{
 		Use:   "launch [launch-id]",
-		Short: "Launch a network as a coordinator",
-		Args:  cobra.ExactArgs(1),
-		RunE:  networkChainLaunchHandler,
+		Short: "Announce the launch a chain",
```

Comment:
`Trigger` is IMO more appropriate than `Announce` because it's not only about communication, the command also perform on-chain state change, setting genesis time and freezing requests

Reply:
https://github.com/ignite/cli/pull/3096/commits/bb3d8008a2c17592bdf1f79da517e70694e194ff